internal/pkg/util/util.go

```diff
@@ -97,8 +102,16 @@ func CopyFiles(source string, dest string) error {
 
 		if info.IsDir() {
 			wwlog.Printf(wwlog.DEBUG, "Creating directory: %s\n", location)
+			info, err := os.Stat(source)
+			if err != nil {
+				return err
+			}
 
-			err := os.MkdirAll(path.Join(dest, location), info.Mode())
+			err = os.MkdirAll(path.Join(dest, location), info.Mode())
+			if err != nil {
+				return err
+			}
+			err = CopyUIDGID(source,dest)
```

Comment:
Did you run this through `gofmt` or fix the formatting by hand?

While this is obviously not a big deal, `gofmt` would have fixed this so there was an appropriate space between `(source, dest)`.